servicetalk-buffer-netty/build.gradle

```diff
@@ -17,17 +17,16 @@
 apply plugin: "io.servicetalk.servicetalk-gradle-plugin-internal-library"
 
 dependencies {
-  api platform(project(":servicetalk-dependencies"))
-  testImplementation enforcedPlatform("org.junit:junit-bom:$junit5Version")
-
+  api platform("io.netty:netty-bom:$nettyVersion")
```

Comment:
what's the benefit of using the netty-bom vs. explicitly using the version for each of the netty dependencies? (thought: keeps all dependency lines/versions consistent in the approach)